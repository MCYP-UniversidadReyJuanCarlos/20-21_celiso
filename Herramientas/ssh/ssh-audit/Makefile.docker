VERSION = $(shell grep VERSION src/ssh_audit/globals.py | grep -E -o "'(v.*)'" | tr -d "'")
ifeq ($(VERSION),)
  $(error "could not determine version!")
endif

all:
	docker build -t positronsecurity/ssh-audit:${VERSION} .
	docker tag positronsecurity/ssh-audit:${VERSION} positronsecurity/ssh-audit:latest

upload:
	docker login
	docker push positronsecurity/ssh-audit:${VERSION}
	docker push positronsecurity/ssh-audit:latest
