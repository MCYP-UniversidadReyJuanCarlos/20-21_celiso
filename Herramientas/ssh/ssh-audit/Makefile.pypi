all:
	rm -rf /tmp/pypi_upload
	virtualenv -p /usr/bin/python3 /tmp/pypi_upload/
	cp -R src /tmp/pypi_upload/
	cp setup.py setup.cfg README.md LICENSE /tmp/pypi_upload/
	/bin/bash -c "pushd /tmp/pypi_upload/; source bin/activate; pip3 install setuptools twine; python3 setup.py sdist bdist_wheel"

uploadtest:
	/bin/bash -c "pushd /tmp/pypi_upload; source bin/activate; twine upload --repository-url https://test.pypi.org/legacy/ /tmp/pypi_upload/dist/*"

uploadprod:
	/bin/bash -c "pushd /tmp/pypi_upload; source bin/activate; twine upload /tmp/pypi_upload/dist/*"

clean:
	rm -rf /tmp/pypi_upload/
