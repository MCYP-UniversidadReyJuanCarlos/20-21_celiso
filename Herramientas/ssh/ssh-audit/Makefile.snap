all:
	echo -e "\n\nDid you remember to bump the version number in snapcraft.yaml?\n\n"
	snapcraft --use-lxd

clean:
	rm -rf parts/ prime/ snap/ stage/ build/ dist/ src/*.egg-info/ ssh-audit*.snap
